openedx/core/djangoapps/user_api/models.py

```diff
@@ -13,6 +17,8 @@
 # create an alias in "user_api".
 from student.models import UserProfile, Registration, PendingEmailChange  # pylint: disable=unused-import
 
+USER_PREFERENCE_TABLE_NAME = "user_api_userpreference"
```

Comment:
Must: Should be able to get this from the event information.

sender._meta.db_table